### PR TITLE
Slides for November 2019

### PR DESCRIPTION
**Pull request by adamchainz**

### CODE REVIEW

**Comment by adamchainz** [comment]
Just did this to make sure the new **POLLEN**  logo is in

File: _layouts/organizers.html [hunk 1]
```diff
@@ -33,7 +33,9 @@
     <body>
         <div class="reveal">
             <div class="slides">
-                {{ content }}
+                <section>
```
Comment (marcoala) [inline _layouts/organizers.html]: This change makes the slides vertical, so we can use a clicker to navigate them

File: organizers/2019-11-26.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    <p>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+      <li><strong>Festicket</strong></li>
+      <li><strong>Pollen (formerly Verve)</strong></li>
+      <li><strong>Redis</strong></li>
```
Comment (marcoala) [inline organizers/2019-11-26.html]: I will mention Redis even if the event is in the past.

File: _layouts/organizers.html [hunk 3]
```diff
@@ -33,7 +33,9 @@
...
+                    {{ content }}
+                </section>
```
Comment (adamchainz) [inline _layouts/organizers.html]: why? each section is a slide, doesn't seem useful to put a slide around it?

File: organizers/2019-10-07.html [hunk 4]
```diff
@@ -26,12 +26,9 @@ <h2>😃 Be Awesome to Each Other </h2>
     <h1>Sponsorship</h1>
     <p>Tonight's meetup is kindly sponsored by:</p>
     <ul>
-      <li><strong>Elastic</strong></li>
       <li><strong>Festicket</strong></li>
-      <li><strong>Mixcloud</strong></li>
       <li><strong>Time Out</strong></li>
       <li><strong>Verve</strong></li>
-      <li><strong>Skills Matter</strong></li>
```
Comment (adamchainz) [inline organizers/2019-10-07.html]: we don't need to remove the sponsors from October do we?

File: _layouts/organizers.html (hunk 1 above)
Comment (adamchainz) [reply _layouts/organizers.html]: Oh right

File: organizers/2019-11-26.html (hunk 2 above)
Comment (adamchainz) [reply organizers/2019-11-26.html]: 👍

File: _layouts/organizers.html (hunk 3 above)
Comment (adamchainz) [reply _layouts/organizers.html]: to make them vertical as @marcoala put

**Pull request merged**